packages/cli/src/utils/is-blitz-root.ts

```diff
@@ -0,0 +1,70 @@
+import {readJSON} from 'fs-extra'
+import pkgDir from 'pkg-dir'
+import {resolve} from 'path'
+
+export enum IsBlitzRootError {
+  notBlitz,
+  notRoot,
+  badPackageJson,
+}
+
+const checkParent = async (): Promise<false | number> => {
+  const rootDir = await pkgDir('./')
+
+  if (rootDir) {
+    const file = await readJSON(resolve(rootDir, 'package.json'))
+
+    if (file && Object.keys(file.dependencies || {}).includes('blitz')) {
+      return process.cwd().slice(rootDir.length).split('/').length - 1
+    }
+  }
+
+  return false
+}
+
+/**
+ * @name isBlitzRoot
+ * @returns IsBlitzRootError enum
+ * notBlitz -> when can't find package.json in current folder and first found in parent
+ *             doesn't have blitz in dependencies
+ * notRoot -> if in a nested folder of blitz project (found blitz as depend in a parent package.json)
+ * badPackageJson -> an error occurred while reading local package.json
+ */
+
+const isBlitzRoot = async (): Promise<{err: boolean; message?: IsBlitzRootError; depth?: number}> => {
+  try {
```

Comment:
i wonder if this check is a bit complex and instead if we can just check for the existence of a `blitz.config.js`?

Reply:
As @flybayer said, we're not requiring `blitz.config.js` nor `next.config.js`, so we should also be able to detect vanilla apps without those files

Reply:
Oops missed that. Disregard then